workflow/rules/structure_learning_algorithms/psilearner/schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "description": "Psi-learner R package",
+    "title": "prsi_learner",
+    "type": "array",
+    "items": {
+        "title": "",
+        "description": "",
+        "type": "object",
+        "properties": {
+            "id": {
+                "type": "string",
+                "description": "Unique identifier"
+            },
+            
+            "timeout": {
+                "$ref": "../../../schemas/definitions.schema.json#/definitions/flexnonnegnumnull"
+            },
+            "alpha1": {
+                "description": "The significance level of correlation screening. In general, a high significance level of correlation screening will lead to a slightly large separator set S_{ij}.",
+                "$ref": "../../../schemas/definitions.schema.json#/definitions/flexprob",
+                "default": 0.05
+            },
+            "alpha2": {
+                "description" : "The significance level of ψ screening.",
+                "$ref": "../../../schemas/definitions.schema.json#/definitions/flexprob",
+                "default": 0.05
+            },
+            "grid" : {
+                "description": "The number of components for the ψ scores. The default value is 2.",
```

Comment:
Maybe e.g. the text _The default value is 2_ can be omitted as it is already default since it set to this value in the example.

Reply:
well the example doesn't say what a default value is, no. Maybe we should keep it. What do you think?